.travis.yml

```diff
@@ -2,19 +2,32 @@ dist: focal
 services:
   - mysql
 language: ruby
-env: CODECLIMATE_REPO_TOKEN=b1b9b856c0aa49c7733aedea0b460b1c118c2ff1296db77bdcf762e6e8ff91c5
+cache: bundler
+env:
+  global:
+    - CC_TEST_REPORTER_ID=b1b9b856c0aa49c7733aedea0b460b1c118c2ff1296db77bdcf762e6e8ff91c5
 before_install:
   - yes | gem update --system --force
   - gem install bundler
-before_script:
-  - cp config/database.yml.example config/database.yml
-  - bundle exec rake db:reset
   - nvm install 'lts/*'
   - nvm use 'lts/*'
+install:
+  - bundle config set --local deployment 'true'
+  - bundle install --jobs=3 --retry=3
```

Comment:
What's the max limit for concurrent jobs on Travis? Could we go higher than 3?(not that we need to)

Reply:
It has to do with the number of CPU cores, setting this higher than N-1 won't offer any improvement. I don't know for sure what the limit is, but this is what [Travis does in their default config](https://docs.travis-ci.com/user/languages/ruby/#dependency-management).